qt/mainwindow.cpp

```diff
@@ -218,6 +220,9 @@ MainWindow::MainWindow(Framework & framework,
   m_pDrawWidget->UpdateAfterSettingsChanged();
 
   RoutingSettings::LoadSession(m_pDrawWidget->GetFramework());
+
+  new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_Q), this, SLOT(close()));
```

Comment:
Let's move this shortcut to L183, after menu items. And add hotkeys for other actions too.

Reply:
Thank you for the suggestion. I have actually simplified it and added the shortcut to the `addAction()` call itself. That has the advantage of showing the shortcut in the menu. I have added shortcuts to the rest of the action as suggested.
Additionally I have rearranged the menu entries to something that looks to me like a more natural order, but that's of course a matter of taste, so I can revert the order if you want to keep the current order.
This is how it looks after my changes:
![Organic Maps_002](https://github.com/organicmaps/organicmaps/assets/17937361/da292aac-0b75-4600-8a5e-f6c8fd094265)